facebook/OpenBIC
# fby3.5: gl: Add gpio table for Great Lakes BIC

Summary:
- Configure Great Lakes GPIOs following the hardware design(GPIO table: 20221104).
- Set BIC_READY to active after BIC firmware is ready. So that, BMC can access BIC through IPMB.

Test Plan:
- Build code :PASS
- Access to BIC console(UART) : PASS
- Check the I2C communication between BMC and BIC(IPMB) : PASS
- Check GPIO list : PASS

Test Log:
1. Build code #west build -p auto -b ast1030_evb meta-facebook/yv35-gl/ ``` [288/295] Linking C executable zephyr/zephyr_prebuilt.elf

    [295/295] Linking C executable zephyr/Y35BGL.elf
    Memory region         Used Size  Region Size  %age Used
            SRAM_NC:        152 KB       256 KB     59.38%
            FLASH:          0 GB         0 GB
                SRAM:        443 KB       512 KB     86.52%
            IDT_LIST:          0 GB         2 KB      0.00%

2. Access to BIC console root@bmc-oob:~# sol-util slot4 sb bic ------------------TERMINAL MULTIPLEXER---------------------
    CTRL-l ?   : Display help message.
    CTRL-l x : Terminate the connection.
    /var/log/mTerm_slot4.log : Log location
    CTRL-l + b : Send Break

    -----------------------------------------------------------
    uart:~$

3. BMC to BIC : root@bmc-oob:~# bic-util slot2 0x18 0x1 00 80 01 00 02 BF 15 A0 00 00 00 00 00 00 00

4. Check GPIO list : uart:~$ platform gpio list_all
    [0  ] HSC_TYPE                           : OD  | input (I) | 1(1)
    [1  ] H_CPU_MON_FAIL_LVC3_N              : OD  | input (I) | 0(0)
    [2  ] FM_SLPS3_LVC3_N                    : OD  | input (I) | 1(1)
    [3  ] IRQ_BMC_NMI_R_N                    : PP  | output(O) | 1(1)
    ```
    [101] AUTH_PRSNT_N                       : OD  | input (I) | 0(0)
